test-cases/longevity/longevity-large-partition-8h.yaml

```diff
@@ -68,4 +68,4 @@ data_validation: |
   primary_key_column: "pk"
   max_partitions_in_test_table: 400
   partition_range_with_data_validation: 0-100
-  limit_rows_number: 10000
+  limit_rows_number: 5555
```

Comment:
no need for this value at all (copy-paste issue). i'll remove it in next PR.

Reply:
Care to explain, why it's not needed?

Reply:
@fruch , so it'll simply use the default (no limit). please see in documentation: https://github.com/scylladb/scylla-cluster-tests/blob/7bb11a2a3eae4b94c7b96928aaec3e6982793d10/sdcm/utils/database_query_utils.py#L46

Reply:
I read it 3 times, but I still don't understand why one case or then would want to set this number.
it's name doesn't tell me anything about it's function/reason.

if if something is wrong with any of the test case in this regards, please open PRs to fix those, no need to wait for some other unrelated fixes for that.

anyhow the main issue I have with those validation that upon failure they kill the nemesis thread.

Reply:
> @fruch , don't you think this validation error is severe enough to kill nemesis thread?
> it means some partition rows are unexpectedly missing.

I don't know enough what is the logic of this check to say.
But one should prove it on every cases it was introduced.

And if it was decided to stop the test, raise a critical error, not just silently kill the nemesis thread.

> 
> As for `limit_rows_number` - the main reason Roy requested it is to avoid choking the system by running a query of, for example, 10M rows, every nemesis.